src/git.rs

```diff
@@ -225,9 +226,23 @@ impl GitStatus {
 }
 
 impl GitStatusItem {
+    pub fn new(file: String) -> Self {
+        GitStatusItem {
+            file,
+            staged: None,
+            unstaged: None,
+        }
+    }
     pub fn file(&self) -> &str {
```

Comment:
And as a side note, how would you feel about changing "file" to "file_name"? I feel that it would be a better descriptor, but maybe not.

Reply:
I agree.